File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// YesAsAService SDK exists test

require_once __DIR__ . '/../yesasaservice_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = YesAsAServiceSDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}

File: rb/test/exists_test.rb
# YesAsAService SDK exists test

require "minitest/autorun"
require_relative "../YesAsAService_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = YesAsAServiceSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
